README.txt

```diff
@@ -1,110 +1,110 @@
+Requires at least: 5.0
+Tested up to: 5.3
+Requires PHP: 7.2
+Stable tag: 1.1.0
+License: GNU General Public License v2.0, MIT License
+
+Improve search on your site. Autocomplete is included, along with full control over look, feel and relevance.
+
+== Description ==
+
+Improve search on your site. Autocomplete is included, along with full control over look, feel and relevance.
+
+= Features =
+* Relevant search results in milliseconds
+* Native typo-tolerance
+* Is language-agnostic
+* Offers hooks and filters for easy customization
+* Built by developers, for developers
+
+This plugin requires API keys from [Algolia](https://www.algolia.com/). API keys are free for small personal projects and non-commercial use. Learn more about [commercial use pricing](https://www.algolia.com/pricing/).
+
+= Links =
+* [WebDevStudios](https://webdevstudios.com)
+* [Algolia](https://algolia.com)
+* [Documentation](https://community.algolia.com/wordpress/configuration.html)
+* [Support](https://github.com/WebDevStudios/wp-search-with-algolia/issues)
+
+*This plugin is a derivative work of the code from the [Search by Algolia – Instant & Relevant results](https://wordpress.org/plugins/search-by-algolia-instant-relevant-results/) plugin for WordPress, which is licensed under the GPLv2.*
+
+== Installation ==
+
+**If you have the *Search by Algolia – Instant & Relevant results* plugin installed, please deactivate it first.**
+
+From your WordPress dashboard:
+
+1. **Visit** Plugins > Add New
+2. **Search** for "WP Search with Algolia"
+3. **Activate** WP Search with Algolia from your Plugins page
+4. **Click** on the new menu item "Algolia Search" and enter your API keys
+5. **Read** the step by step [configuration guide](https://community.algolia.com/wordpress/configuration.html)
+
+== Frequently Asked Questions ==
+
+= Is this plugin a fork? =
+
+Yes. The Algolia Team **[no longer supports their original plugin](https://community.algolia.com/wordpress/)**. The engineering team at WebDevStudios has forked the original plugin, and is now maintaining it.
+
+= Should I switch to this plugin? =
+
+Yes. Because Algolia no longer supports their plugin, you will no longer receive updates. WebDevStudios uses Algolia on many of it's projects, and is committed to maintaining this plugin for years to come.
+
+= How do I switch from the "Search by Algolia – Instant & Relevant results" plugin? =
+
+1. **Deactivate** the *Search by Algolia – Instant & Relevant results* plugin
+2. **Follow** the [installation instructions](https://wordpress.org/plugins/wp-search-with-algolia/#installation)
+3. **Activate** *WP Search with Algolia*
+4. **Check** for your API Keys. They should already be there, if not, enter them and then save settings
+5. **Delete** the *Search by Algolia – Instant & Relevant results* plugin
+
+= What are the minimum requirements? =
+
+* Requires WordPress 4.4+

```

Comment:
@richaber Do we need to update these as well?

Reply:
Good call, missed that.

Reply:
Commit 567a741 Update Min Requirements README verbiage to WP 5.0 and PHP 7.2